libs/utils/test.py

```diff
@@ -87,6 +87,14 @@ def runExperiments(cls):
         test_env = TestEnv(test_conf=cls._getTestConf())
 
         experiments_conf = cls._getExperimentsConf(test_env)
+
+        if ITERATIONS_FROM_CMDLINE:
```

Comment:
Because I want it to be as clear as possible that it's a different source for the value.. the mechanism I've implemented here pretty convoluted so I think long descriptive names are gonna be helpful..